### PR TITLE
[Feature] add state storage in part component

### PR DESCRIPTION
**Pull request by dtiwarATS**
-- added state storage for all the Janus part components created so far in the Torus
-- added missing CSS file for Carousel components.

### CODE REVIEW

File: assets/src/utils/helpers.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// parse value and return accurate boolean
+// returns boolean values for both numbers and strings
+export const parseBool = (val: any) => {
```
Comment (bsparks) [inline assets/src/utils/helpers.ts:3]: we already have something here https://github.com/Simon-Initiative/oli-torus/blob/master/assets/src/utils/common.ts#L49 perhaps there is a new case?

File: assets/src/components/parts/types/parts.ts [hunk 2]
```diff
@@ -12,4 +12,11 @@ export interface JanusAbsolutePositioned extends CustomProperties {
   z?: number;
   width?: number;
   height?: number;
-}
\ No newline at end of file
+}
+
+export interface StateVariable {
```
Comment (bsparks) [inline assets/src/components/parts/types/parts.ts:17]: I think we will replace this one with capivariable

File: assets/src/components/parts/janus-Carousel/Carousel.tsx [hunk 3]
```diff
@@ -56,6 +63,32 @@ const Carousel: React.FC<any> = (props) => {
       : `calc(${height} - ${MAGIC_NUMBER}px)`,
   };
 
+  useEffect(() => {
+    props.onReady({
+      activityId: `${id}`,
+      partResponses: [
```
Comment (bsparks) [inline assets/src/components/parts/janus-Carousel/Carousel.tsx:69]: in my latest branch (not yet master) I've been doing it like this:
https://github.com/Simon-Initiative/oli-torus/blob/feature/adaptive-activity-3/assets/src/components/parts/janus-input-text/InputText.tsx#L36

**Comment by dtiwarATS** [comment]
Closing this PR as I will be created another branch based of "feature/adaptive-activity-3" branch and create another PR

**Pull request closed without merging**